.npmignore

```diff
@@ -1,19 +0,0 @@
-# Files
```

Comment:
this blacklist is not needed since we're using the whitelist in `package.json/files`